ambari/administration/admin-tasks.rst

```diff
@@ -0,0 +1,109 @@
+Ambari Admin Tasks
+==================
+
+An "Ambari Admin" has administrator (or super-user) privilege. When logged into Ambari with the "Ambari Admin" privilege, you can:
+
+* :ref:`Change the Administrator Account Password<ambari-change-admin-password>`
+* :ref:`Create a cluster<ambari-creating-cluster>`
+* :ref:`Set access permissions for an existing cluster<ambari-cluster-permission>`
+* :doc:`Create, delete, and edit view instances</ambari/administration/views>`
+* :ref:`Manage permissions for view instances<ambari-setting-view-permissions>`
+* :doc:`Create, edit, and delete users and user groups</ambari/administration/user-group>`
```

Comment:
_Managing Users and Groups_ vs _Create, edit, and delete users and user groups_, Using same text will improve user reading.